.github/workflows/deploy.yaml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   deploy:
+    if: $OLD_ECS_DEPLOY == 'true'
```

Comment:
reviewer note: once we merge back to main we don't want to deploy anything so added this check